mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -210,28 +209,27 @@ public void listUsers(ListUsersRequest request, StreamObserver<ListUsersResponse
     public void getRights(GetRightsRequest request, StreamObserver<GetRightsResponse> responseObserver) {
         String actorCrn = request.getActorCrn();
         String accountId = Crn.fromString(actorCrn).getAccountId();
-        List<Group> groups = List.copyOf(getOrCreateGroups(accountId));
+        List<Group> groups = List.copyOf(mockGroupManagementService.getOrCreateGroups(accountId));
         Group group = groups.get(FIRST_GROUP);
         PolicyStatement policyStatement = PolicyStatement.newBuilder()
                 .addRight(ALL_RIGHTS_AND_RESOURCES)
                 .addResource(ALL_RIGHTS_AND_RESOURCES)
                 .build();
         PolicyDefinition policyDefinition = PolicyDefinition.newBuilder().addStatement(policyStatement).build();
         Policy powerUserPolicy = Policy.newBuilder()
-                .setCrn(createCrn(ACCOUNT_ID_ALTUS, Crn.Service.IAM, Crn.ResourceType.POLICY, "PowerUserPolicy").toString())
+                .setCrn(mockCrnService.createCrn(ACCOUNT_ID_ALTUS, Crn.Service.IAM, Crn.ResourceType.POLICY, "PowerUserPolicy").toString())
                 .setCreationDateMs(CREATION_DATE_MS)
                 .setPolicyDefinition(policyDefinition)
                 .build();
         Role powerUserRole = Role.newBuilder()
-                .setCrn(createCrn(ACCOUNT_ID_ALTUS, Crn.Service.IAM, Crn.ResourceType.ROLE, "PowerUser").toString())
+                .setCrn(mockCrnService.createCrn(ACCOUNT_ID_ALTUS, Crn.Service.IAM, Crn.ResourceType.ROLE, "PowerUser").toString())
                 .setCreationDateMs(CREATION_DATE_MS)
                 .addPolicy(powerUserPolicy)
                 .build();
         RoleAssignment roleAssignment = RoleAssignment.newBuilder().setRole(powerUserRole).build();
         GetRightsResponse.Builder responseBuilder = GetRightsResponse.newBuilder()
                 .addGroupCrn(group.getCrn())
-                .addRoleAssignment(roleAssignment)
-                .addWorkloadAdministrationGroupName("mockworkloadadministrationgroup0");
+                .addRoleAssignment(roleAssignment);
```

Comment:
we need to have some workload administration group returned in the GetRights call because that is how user sync determines group membership.

Reply:
right now anything is fine from the mock. your suggested virtual group is fine. I just want to make sure there's something there for user sync to read and push to the ipa server.

Reply:
Okay, thanks, will be fixed immediately.